extensions/sample-command/build.gradle

```diff
@@ -21,8 +21,13 @@ checkSourceFormatting {
 }
 
 dependencies {
+	compile project(":cli")
```

Comment:
`compileOnly`

Reply:
Unfortunately this change causes the tests to fail, maybe we can discuss later today. All other change requests have been implemented successfully.